### PR TITLE
Fix next page logic - FLAT-2702

### PR DESCRIPTION
**Pull request by ronanbarrett**
Forked the repo as it is not maintained anymore. The paging logic was flawed so I fixed it.

### CODE REVIEW

File: ghbackup/fetch.go [hunk 1]
```diff
@@ -126,11 +126,17 @@ func getNextURL(header http.Header) string {
 	if len(parts) == 0 {
 		return ""
 	}
-	firstLink := parts[0]
-	if !strings.Contains(firstLink, "rel=\"next\"") {
+	var nextLink string
+	for _, v := range parts {
+		if strings.Contains(v, "rel=\"next\"") {
+			nextLink = strings.TrimSpace(v)
```
Comment (jeespers) [inline ghbackup/fetch.go:132]: **Question [clarification]:** 

Do we want to break here once we find a link starting containing `"rel=\"next\"" ` or is the last link with the string matching we're interested in?

Comment (jeespers) [reply ghbackup/fetch.go:132]: Basically, I wonder if there can there be a bunch of links containing the string `"rel=\"next\""` or always just one?

Comment (ronanbarrett) [reply ghbackup/fetch.go:132]: Yeah there should be just one "next". Added a break.

Comment (ronanbarrett) [reply ghbackup/fetch.go:132]: Am gonna merge this now.

**Pull request merged**